### PR TITLE
[Compiler] move expressions out of convert file.

### PR DESCRIPTION
**Pull request by Jim8y**
This pr only move expression convert into seperate files for the ease of debugging and managing.

Not a single line of the code is changed. 

how to review?
Copy methods in each expression file and search in the original method convert.

### CODE REVIEW

**Review by cschuchardt88 — APPROVED** [review]

**Comment by Jim8y** [comment]
@shargon

**Comment by Jim8y** [comment]
@superboyiii

File: src/Neo.Compiler.CSharp/MethodConvert/Expression/BinaryExpression.cs [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+extern alias scfx;
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Neo.VM;
+
+namespace Neo.Compiler;
+partial class MethodConvert
+{
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/Expression/BinaryExpression.cs:20]:
```suggestion

partial class MethodConvert
{
```

File: src/Neo.Compiler.CSharp/MethodConvert/Expression/CheckedExpression.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+partial class MethodConvert
+{
+    private void ConvertCheckedExpression(SemanticModel model, CheckedExpressionSyntax expression)
+    {
+        _checkedStack.Push(expression.Keyword.IsKind(SyntaxKind.CheckedKeyword));
+        ConvertExpression(model, expression.Expression);
+        _checkedStack.Pop();
+    }
+
+}
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/Expression/CheckedExpression.cs:28]:
```suggestion
}
```

File: src/Neo.Compiler.CSharp/MethodConvert/Expression/UnaryExpression.cs [hunk 3]
```diff
@@ -0,0 +1,476 @@
...
+        AddInstruction(operatorToken.ValueText switch
+        {
+            "++" => OpCode.INC,
+            "--" => OpCode.DEC,
+            _ => throw new CompilationException(operatorToken, DiagnosticId.SyntaxNotSupported, $"Unsupported operator: {operatorToken}")
+        });
+        if (typeSymbol != null) EnsureIntegerInRange(typeSymbol);
+    }
+
+}
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/Expression/UnaryExpression.cs:476]:
```suggestion
}
```

**Review by shargon — APPROVED** [review]

**Pull request merged**